includes/features/onboarding/class-wc-admin-onboarding.php

```diff
@@ -26,6 +26,43 @@ public static function get_instance() {
 		}
 		return self::$instance;
 	}
+
+	/**
+	 * Hook into WooCommerce.
+	 */
+	public function __construct() {
+		add_action( 'woocommerce_components_settings', array( $this, 'component_settings' ), 20 ); // Run after WC_Admin_Loader.
+	}
+
+	/**
+	 * Add alert count to the component settings.
+	 *
+	 * @param array $settings Component settings.
+	 */
+	public function component_settings( $settings ) {
+		$redirect_url = esc_url_raw(
+			add_query_arg(
+				array(
+					'page' => 'wc-admin',
+				),
+				admin_url( 'admin.php' )
+			) . '#/?step=details'
+		);
+
+		$settings['jetpackConnectUrl'] = Jetpack::init()->build_connect_url( true, $redirect_url, 'woocommerce-setup-wizard' );
```

Comment:
😆. This was one of the latest pieces I did and I guess I didn't re-test with Jetpack disabled. We'll have to roll our own function.

Reply:
I ended up just generating this on demand, once the plugin has been activated. There's a lot of logic in the connect function / it generates nonces, so that seems like the best approach.